ajax/libs/twbuttons/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "twbuttons",
+  "filename": "dist/twbuttons.css",
+  "description": "alexwolfe/Buttons for Twitter Bootstrap 3",
+  "license": "MIT",
+  "keywords": [
+    "buttons",
+    "less",
+    "css",
+    "css buttons",
+    "less buttons",
+    "bootstrap"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/onokumus/twbuttons.git"
+  },
+  "homepage": "https://onokumus.github.io/twbuttons/",
+  "author": "Osman Nuri Okumuş <onokumus@gmail.com> (https://github.com/onokumus)",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/onokumus/twbuttons.git",
+    "basePath": "dist"
```

Comment:
missing `}` caused syntax error, please verify the package.json syntax first: http://jsonlint.com/

@pvnr0082t I remember that you were working on the syntax error checking in the `npm test`? Any updates?